xbmc/platform/posix/utils/PosixInterfaceForCLog.h

```diff
@@ -17,7 +17,7 @@ class CPosixInterfaceForCLog : public IPlatformLog
   virtual ~CPosixInterfaceForCLog() override = default;
 
   spdlog_filename_t GetLogFilename(const std::string& filename) const override { return filename; }
-  void AddSinks(
+  virtual void AddSinks(
```

Comment:
Personally I don't like specifying `virtual` on virtual functions in derived classes. The method has already been declared `virtual` in the base class (in this case `IPlatformLog`) so no need to do it again. Furthermore you already have the `override` keyword which also implies `virtual`.

Reply:
The team agreed to have automated formatting remove the virtual on derived classes, so we should probably do the same.

* The argument for this is that `override` exists and `virtual` is semi-redundant.
* The argument against is that it's easier to notice a word at the beginning of a line rather than at the end. Ignoring parts of lines isn't a readability problem though, it's a lazy reading problem.

Subjective readability is weighed against the expectation of being read, and expectations are stronger than opinions. When I realized this I became OK with the team decision :)

Reply:
dropped